ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -516,3 +516,145 @@ TargetUrl,string,Optional,FileEvent,,,
+ParentProcessGuid,string,Optional,ProcessEvent,string,,
+ParentProcessId,string,Mandatory,ProcessEvent,string,,
+ParentProcessIMPHASH,string,Optional,ProcessEvent,string,,
+ParentProcessInjectedAddress,string,Optional,ProcessEvent,string,,
+ParentProcessIntegrityLevel,string,Optional,ProcessEvent,string,,
+ParentProcessIsHidden,Boolean,Optional,ProcessEvent,Boolean,,
+ParentProcessMD5,string,Optional,ProcessEvent,MD5,,
+ParentProcessName,string,Optional,ProcessEvent,string,,
+ParentProcessSHA1,string,Optional,ProcessEvent,SHA1,,
+ParentProcessSHA256,string,Optional,ProcessEvent,SHA256,,
+ParentProcessSHA512,string,Optional,ProcessEvent,SHA512,,
+ParentProcessTokenElevation,string,Optional,ProcessEvent,string,,
+Process,string,Alias,ProcessEvent,,,TargetProcessName
+TargetProcessCommandLine,string,Mandatory,ProcessEvent,string,,
+TargetProcessCreationTime,datetime,Mandatory,ProcessEvent,DateTime,,
+TargetProcessCurrentDirectory,string,Optional,ProcessEvent,string,,
+TargetProcessFileCompany,string,Optional,ProcessEvent,string,,
+TargetProcessFileDescription,string,Optional,ProcessEvent,string,,
+TargetProcessFileInternalName,string,Optional,ProcessEvent,string,,
+TargetProcessFileOriginalName,string,Optional,ProcessEvent,string,,
+TargetProcessFileProduct,string,Optional,ProcessEvent,string,,
+TargetProcessFileSize,string,Optional,ProcessEvent,string,,
+TargetProcessFileVersion,string,Optional,ProcessEvent,string,,
+TargetProcessGuid,string,Optional,ProcessEvent,string,,
+TargetProcessId,string,Mandatory,ProcessEvent,string,,
+TargetProcessIMPHASH,string,Optional,ProcessEvent,string,,
+TargetProcessInjectedAddress,string,Optional,ProcessEvent,string,,
+TargetProcessIntegrityLevel,string,Optional,ProcessEvent,string,,
+TargetProcessIsHidden,Boolean,Optional,ProcessEvent,Boolean,,
+TargetProcessMD5,string,Optional,ProcessEvent,MD5,,
+TargetProcessName,string,Mandatory,ProcessEvent,string,,
+TargetProcessSHA1,string,Optional,ProcessEvent,SHA1,,
+TargetProcessSHA256,string,Optional,ProcessEvent,SHA256,,
+TargetProcessSHA512,string,Optional,ProcessEvent,SHA512,,
+TargetProcessTokenElevation,string,Optional,ProcessEvent,string,,
+TargetUserId,string,Recommended,ProcessEvent,string,,
+TargetUserIdType,string,Recommended,ProcessEvent,string,,
+TargetUsername,string,Mandatory for process create events.,ProcessEvent,string,,
+TargetUsernameType,string,Mandatory for process create events.,ProcessEvent,Enumerated,UPN|Windows|DN|Simple,
+TargetUserSessionId,string,Optional,ProcessEvent,string,,
+TimeGenerated,datetime,Mandatory,RegistryEvent,,,
+_ResourceId,string,Mandatory,RegistryEvent,,,
+Type,string,Mandatory,RegistryEvent,,,
+EventMessage,string,Optional,RegistryEvent,,,
+AdditionalFields,dynamic,Optional,RegistryEvent,,,
+Dvc,string,Mandatory,RegistryEvent,,,
+DvcHostname,string,Recommended,RegistryEvent,,,
+DvcId,string,Optional,RegistryEvent,,,
+DvcIpAddr,string,Recommended,RegistryEvent,,,
```

Comment:
You should the relevant Logical types (IP address, Hostname etc.) see the schemas I created or the data tester script.